modules/core/src/lib/layer-manager.js

```diff
@@ -311,11 +312,13 @@ export default class LayerManager {
         }
 
         if (!oldLayer) {
-          this._initializeLayer(newLayer);
+          const err = this._initializeLayer(newLayer);
```

Comment:
@ibgreen Were these errors intentionally ignored? They were blocking tests from capturing layer update errors.

Reply:
> Were these errors intentionally ignored?

That would have been a long time ago... I don't have any recollection but it is certainly conceivable that errors during initialization interfered with deck.gl state keeping and that it was intentionally left out. 

This code looks pretty defensive though, but I guess we'll find out if there was a reason for it now that we have made the change :)